back/engines/commercial/admin_api/app/controllers/admin_api/tenants_controller.rb

```diff
@@ -6,9 +6,13 @@ class TenantsController < AdminApiController
     skip_around_action :switch_tenant
 
     def index
-      tenants = Tenant.not_deleted.order(name: :asc)
+      tenants = Tenant.not_deleted
       # Call #to_json explicitly, otherwise 'data' is added as root.
-      render json: serialize_tenants(tenants).to_json
+      tenants_json = Rails.cache.fetch(tenants.cache_key_with_version, expires_in: 1.hour) do
```

Comment:
According to [this Metabase query](https://metabase.hq.citizenlab.co/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJTRUxFQ1RcbiAgXCJwcm9kdWN0aW9uX2NsMl9iYWNrX3Jhd1wiLlwiY2wyX3Jhd19hcHBfY29uZmlndXJhdGlvbnNcIi5cImlkXCIgQVMgXCJpZFwiLFxuICBFWFRSQUNUKEVQT0NIIEZST00gXCJwcm9kdWN0aW9uX2NsMl9iYWNrX3Jhd1wiLlwiY2wyX3Jhd19hcHBfY29uZmlndXJhdGlvbnNcIi5cInVwZGF0ZWRfYXRcIikgQVMgXCJ1cGRhdGVkX2F0XCIsXG4gIEVYVFJBQ1QoRVBPQ0ggRlJPTSBcIlRlbmFudHNcIi5cInVwZGF0ZWRfYXRcIikgQVMgXCJUZW5hbnRzX191cGRhdGVkX2F0XCIsXG4gIEVYVFJBQ1QoRVBPQ0ggRlJPTSBcInByb2R1Y3Rpb25fY2wyX2JhY2tfcmF3XCIuXCJjbDJfcmF3X2FwcF9jb25maWd1cmF0aW9uc1wiLlwidXBkYXRlZF9hdFwiKSAtXG4gIEVYVFJBQ1QoRVBPQ0ggRlJPTSBcIlRlbmFudHNcIi5cInVwZGF0ZWRfYXRcIikgYXMgXCJkaWZmXCJcbkZST01cbiAgXCJwcm9kdWN0aW9uX2NsMl9iYWNrX3Jhd1wiLlwiY2wyX3Jhd19hcHBfY29uZmlndXJhdGlvbnNcIlxuICBMRUZUIEpPSU4gXCJwcm9kdWN0aW9uX2NsMl9iYWNrX3Jhd1wiLlwiY2wyX3Jhd190ZW5hbnRzXCIgQVMgXCJUZW5hbnRzXCIgT04gXCJwcm9kdWN0aW9uX2NsMl9iYWNrX3Jhd1wiLlwiY2wyX3Jhd19hcHBfY29uZmlndXJhdGlvbnNcIi5cInRlbmFudF9pZFwiID0gXCJUZW5hbnRzXCIuXCJpZFwiXG53aGVyZVxuICBFWFRSQUNUKFxuICAgIEVQT0NIXG4gICAgRlJPTVxuICAgICAgXCJwcm9kdWN0aW9uX2NsMl9iYWNrX3Jhd1wiLlwiY2wyX3Jhd19hcHBfY29uZmlndXJhdGlvbnNcIi5cInVwZGF0ZWRfYXRcIlxuICApIDw-IEVYVFJBQ1QoXG4gICAgRVBPQ0hcbiAgICBGUk9NXG4gICAgICBcIlRlbmFudHNcIi5cInVwZGF0ZWRfYXRcIlxuICApIEFORFxuICBFWFRSQUNUKEVQT0NIIEZST00gXCJwcm9kdWN0aW9uX2NsMl9iYWNrX3Jhd1wiLlwiY2wyX3Jhd19hcHBfY29uZmlndXJhdGlvbnNcIi5cInVwZGF0ZWRfYXRcIikgLVxuICBFWFRSQUNUKEVQT0NIIEZST00gXCJUZW5hbnRzXCIuXCJ1cGRhdGVkX2F0XCIpIDw-IDg2NDAwXG5MSU1JVFxuICAxMDQ4NTc1IiwidGVtcGxhdGUtdGFncyI6e30sImNvbGxlY3Rpb24iOiJjbDJfcmF3X2FwcF9jb25maWd1cmF0aW9ucyJ9LCJkYXRhYmFzZSI6M30sImRpc3BsYXkiOiJ0YWJsZSIsInBhcmFtZXRlcnMiOltdLCJ2aXN1YWxpemF0aW9uX3NldHRpbmdzIjp7fX0=), `updated_at` of tenants and app configurations looks different, but I checked two anomalies and one other tenant on the server, and the timestamps were the same.

Reply:
Yep, in principle, that's one of our invariants. If it's not respected, we have a bug.